etc/config-template.toml

```diff
@@ -158,6 +158,18 @@
 ## Otherwise, the OOM problem might occur in TiKV.
 # capacity = "1GB"
 
+[storage.gc]
```

Comment:
I know @MyonKeminta wants to separate GC from storage module, so can we use `[gc]` section direclty？